Form-Controls/README.md

```diff
@@ -30,18 +30,18 @@ Do not write a form action for this project.
 
 Let's write out our testable criteria. Check each one off as you complete it.
 
-- [ ] I have used HTML only.
+- [x] I have used HTML only.
```

Comment:
Can you review the files included in your PR? Currently you have files from the Form-Controls exercise in this PR.